src/styles.js

```diff
@@ -5,14 +5,18 @@ export const GlobalStyle = createGlobalStyle`
 	*::before,
 	*::after {
 		box-sizing: border-box;
+
```

Comment:
Please delete empty lines -  but you can do that in the next pull request 😉

Reply:
Oh boy... those lines are so easy to overlook. 🙈 Thanks for pointing it out. 👍🏼